recipes/qt/5.x.x/conanfile.py

```diff
@@ -1114,7 +1110,10 @@ def _create_plugin(pluginname, libname, plugintype, requires):
                 _create_module("VulkanSupport", ["Core", "Gui"])
 
             if self.options.widgets:
-                _create_module("Widgets", ["Gui"])
+                widget_reqs = ["Gui"]
+                if self.options.with_freetype:
+                    widget_reqs.append("freetype::freetype")
+                _create_module("Widgets", widget_reqs)
```

Comment:
What makes you think freetype is a requirements of Widgets instead of GUI ? freetype is checked by qt in [GUI](https://github.com/qt/qtbase/blob/v5.15.10-lts-lgpl/src/gui/configure.json#L1199) but not in [Widgets](https://github.com/qt/qtbase/blob/v5.15.10-lts-lgpl/src/widgets/configure.json)

Reply:
Freetype is hidden in fontdatabase, plugins.
Their also are reference in platforms/xcb and platforms/minimal plugin, but I don't know why
The only reference to freetype in .pro/.pri for linking with it are:
https://github.com/qt/qtbase/blob/v5.15.10-lts-lgpl/src/platformsupport/fontdatabases/freetype/freetype.pri
https://github.com/qt/qtbase/blob/v5.15.10-lts-lgpl/src/plugins/platforms/xcb/nativepainting/nativepainting.pri
https://github.com/qt/qtbase/blob/v5.15.10-lts-lgpl/src/plugins/platforms/minimal/minimal.pro